### DISCUSSION TITLE
Feature request: Show serial number on taskbar.

### DISCUSSION THREAD

**Discussion by photkey** [valinet/ExplorerPatcher#863] [General]
Windows system comes with the shortcut key Win+number to switch applications quickly and precisely, but if you have to count the serial number of the application each time, it is not very efficient; if you can display the serial number on the taskbar, it will greatly improve the switching speed and is more efficient than Alt+Tab switching.

**Comment by valinet** [comment]
It’s not more efficient than Alt-Tab as they are not comparable. One does a thing, the other something else.

So, suppose we do that, how fo we do it realistically? Every time one presses the Win key, no matter if you want a Win+number shortcut or something else, a number should be overlaid over the taskbar icon…? That’s kind of disturbing, imo. I am curious to see how this would be implemented, from a functionality point of view.

And yeah, it would have been best if the request to start these kind of posts in Discussions was not dismissed.

**Reply by photkey** [reply]
Already found a tool that is compatible with EP. https://ramensoftware.com/7-taskbar-numberer
